smoke-tests/spec/external_dns_helper.rb

```diff
@@ -1,31 +1,32 @@
-def create_ingress
+def create_ingress(namespace, ingress_name, fixture_name)
+  sleep 1
```

Comment:
For the "sleep 1" at the start of methods, please add a comment explaining why it's there (because it looks like it's unnecessary). I'd put something like:

> \# TODO: sleep added to avoid AWS Route53 API throttling errors. Remove once that issue is resolved.